commands/payment_channel_daemon_test.go

```diff
@@ -150,14 +150,17 @@ func TestPaymentChannelVoucherSuccess(t *testing.T) {
 	chanid, _ := rsrc.requirePaymentChannel(ctx, t, channelAmount, channelExpiry)
 
 	voucherAmount := types.NewAttoFILFromFIL(10)
-	voucherValidAt := types.NewBlockHeight(0)
+	voucherValidAt := types.NewBlockHeight(1)
```

Comment:
I noticed that not all the info was being checked as part of debugging the deals show test, and added more asserts.